server/container/guice/blob-export-guice/src/test/java/org/apache/james/modules/LinshareGuiceExtension.java

```diff
@@ -35,6 +36,11 @@ public LinshareGuiceExtension() {
         linshareExtension = new LinshareExtension();
     }
 
+    @BeforeAll
+    public void beforeAll(ExtensionContext extensionContext) {
```

Comment:
you don't need to explicitly declare this method. Junit can do it implicitly basing on the type of LinshareExtension (BeforeAllCallBack)

Reply:
`@BeforeAll` is wrong typed, it should be `@Override`.